java/src/jmri/jmrit/beantable/block/BlockTableDataModel.java

```diff
@@ -64,7 +64,8 @@
     String defaultBlockSpeedText;
     
     public BlockTableDataModel(Manager<Block> mgr){
-        super(mgr);
+        super();
+        setManager(mgr); // for consistency, no local method as 
```

Comment:
I'm not sure I understand this comment: `no local method as`

Reply:
Thanks, incomplete comment!